config/initializers/paperclip.rb

```diff
@@ -27,3 +26,21 @@
 end
 
 Paperclip::Attachment.default_options.update(options)
+
+module PaperclipUrlPatch
+  def url(style_name = default_style, options = {})
+    if options.has_key?(:size)
+      # Paperclipのurlに含まれる:sizeに指定したサイズ値を挿入する
+      # http://stackoverflow.com/questions/4041373/paperclip-custom-path-and-url
+      Paperclip.interpolates :size do |attachment, style|
```

Comment:
Unused block argument - `attachment`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `style`. You can omit all the arguments if you don't care about them.